react-renderer/src/Circle.tsx

```diff
@@ -14,6 +14,14 @@ class Circle extends React.Component<IGPIProps> {
     const strokeAlpha = shape.strokeColor.contents.contents[3];
     const thickness = shape.strokeWidth.contents;
 
+    let dashArray = "7,5";
+    if( "strokeDashArray" in shape )
```

Comment:
Is this the new formatting? Not super standard in JS/TS projects. @maxkrieger any opinions?

Reply:
I just use whatevers default for prettier + ts. I agree this formatting is atypical

Reply:
Oh, this was Keenan's code. I just fixed the formatting.